patroni/ctl.py

```diff
@@ -1372,3 +1373,246 @@ def format_pg_version(version):
+    ``bootsrtap.dcs.postgresql.parameters``). Either the provided DSN or PG env vars will be used for the connection.
+
+    The created configuration contains:
+    - ``scope``: the provided option value. Will be overwritten with the cluster_name GUC value if it is available
+    - ``name``: hostname
+    - ``bootsrtap.dcs``: section with all the parameters (incl. the majority of PG GUCs) set to their default values
+      defined by Patroni and adjusted by the source instances's configuration values.
+    - ``postgresql.parameters``: the source instance's archive_command, restore_command, archive_cleanup_command,
+      recovery_end_command, ssl_passphrase_command, hba_file, ident_file, config_file GUC values
+    - ``postgresql.bin_dir``: path to Postgres binaries gathered from the running instance or, if not available,
+      the value of the --bin-dir option if provided. Otherwise, an empty string.
+    - ``postgresql.datadir``: the value gathered from the corresponding PG GUC
+    - ``postgresql.listen``: source instance's listen_addresses and port GUC values
+    - ``postgresql.connect_address``: if possible, generated from the connection params
+    - ``postgresql.authentication``:
+        - superuser and replication users defined (if possible, usernames are set from the respective Patroni ENV vars,
+          otherwise the default 'postgres' and 'replicator' values are used).
+          If not a sample config, either DSN or PG ENV vars are used to define superuser authentication parameters.
+        - rewind user is defined for a sample config if PG version can be defined and PG version is 11+
+          (if possible, username is set from the respective Patroni ENV var)
+    - ``bootsrtap.dcs.postgresql.use_pg_rewind set to True if PG version is 11+
+    - ``postgresql.pg_hba`` defaults or the lines gathered from the source instance's hba_file
+    - ``postgresql.pg_ident`` the lines gathered from the source instance's ident_file
+
+    :param scope: Scope parameter value to write into the configuration.
+    :param file: Full path to the configuration file to be created (/tmp/patroni.yml by default).
+    :param sample: Optional flag. If set, no source instance will be used - generate config with some sane defaults.
+    :param dsn: Optional DSN string for the local instance to get GUC values from.
+    :param bin_dir: Optional path to Postgres binaries.
+    """
+    from patroni.config import Config
+    from patroni.validator import get_major_version
+
+    no_value_msg = '#FIXME'
+    pg_version = None
+
+    dynamic_config = Config.get_default_config()
+    dynamic_config['postgresql']['parameters'] = dict(dynamic_config['postgresql']['parameters'])
+    config = {
+        'scope': scope or no_value_msg,
+        'name': socket.gethostname(),
+        'bootstrap': {
+            'dcs': dynamic_config
+        },
+        'postgresql': {
+            'parameters': None,
```

Comment:
```suggestion
```

Reply:
I wanted to have it mentioned in the config file even nothing is added there
```
  parameters:
  pg_hba:
   - ...
  ```